### PR TITLE
style: fix import format at pandas/tests/io/formats

### PR DESCRIPTION
**Pull request by streeck**
- [ ] xref #23334 
- [x] passes `git diff master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

**Review by alimcmaster1 — CHANGES_REQUESTED** [review]
Thanks for the PR -Please merge master and look into failing CI :)

**Comment by jreback** [comment]
can you merge master, we just updated the isort cfg

**Comment by streeck** [comment]
Will do later today

File: pandas/tests/io/formats/test_style.py [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+
+import pandas.util._test_decorators as td
+
 import pandas as pd
 from pandas import DataFrame
 import pandas.util.testing as tm
-import pandas.util._test_decorators as td
 
-jinja2 = pytest.importorskip('jinja2')
 from pandas.io.formats.style import Styler, _get_level_lengths  # noqa
```
Comment (streeck) [inline pandas/tests/io/formats/test_style.py]: This is the reason the build is failing, the `jinja2 = pytest.importorskip('jinja2')` should come before.
Should I reorder the lines and add `# isort:skip`?

Comment (alimcmaster1) [reply pandas/tests/io/formats/test_style.py]: I'm wondering why the `importerskip` is needed and why we don't add jinja as a dependency for the relevant builds, for now I think skipping is a fair idea unless anyone has ideas regarding my first point.

**Comment by jbrockmendel** [comment]
Travis failure looks like it’s caused by missing jinja2.  Is there a viable solution to the importorskip?

File: pandas/tests/io/formats/test_style.py (hunk 1 above)
Comment (streeck) [reply pandas/tests/io/formats/test_style.py]: Alright, I'll try with the `isort:skip`

**Comment by jreback** [comment]
can you merge master and fixup

**Comment by jreback** [comment]
@streeck pls merge master

**Comment by datapythonista** [comment]
Closing as stale, @streeck please let us know if you want to continue working on this.

**Pull request closed without merging**